src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/_params.py

```diff
@@ -26,37 +24,60 @@
+        c.argument('username', options_list=('--username', '-u'), help='The username used to log into a container registry')
+        c.argument('password', options_list=('--password', '-p'), help='The password used to log into a container registry')
+
+    with self.argument_context('acr create') as c:
+        c.argument('admin_enabled', nargs='?', required=False, const='true', default='false', help='Indicates whether the admin user is enabled', choices=['true', 'false'])
+
+    with self.argument_context('acr update') as c:
+        c.argument('admin_enabled', help='Indicates whether the admin user is enabled', choices=['true', 'false'])
+
+    with self.argument_context('acr repository delete') as c:
+        c.argument('manifest', nargs='?', required=False, const='', default=None, help='The sha256 based digest of manifest to delete')
+        c.argument('yes', options_list=('--yes', '-y'), action='store_true', help='Do not prompt for confirmation')
+        c.argument('repository', help='The name of the repository to delete.')
+        c.argument('tag', help='The name of tag to delete')
+
+    with self.argument_context('acr repository show-manifests') as c:
+        c.argument('repository', help='The repository to obtain manifests from.')
+
+    with self.argument_context('acr repository show-tags') as c:
+        c.argument('repository', help='The repository to obtain tags from.')
+
+    with self.argument_context('acr create') as c:
+        c.argument('registry_name', completer=None, validator=validate_registry_name)
+        c.argument('deployment_name', arg_type=deployment_name_type, validator=None)
+        c.argument('location', arg_type=get_location_type(self.cli_ctx), validator=get_default_location_from_resource_group)
+
+    with self.argument_context('acr check-name') as c:
+        c.argument('registry_name', completer=None)
+
+    with self.argument_context('acr webhook') as c:
+        c.argument('registry_name', options_list=('--registry', '-r'), help='The name of the container registry. You can configure the default registry name using `az configure --defaults acr=<registry name>`', completer=get_resource_name_completion_list(REGISTRY_RESOURCE_TYPE), configured_default='acr')
+        c.argument('webhook_name', options_list=('--name', '-n'), help='The name of the webhook', completer=get_resource_name_completion_list(WEBHOOK_RESOURCE_TYPE))
+        c.argument('uri', help='The service URI for the webhook to post notifications.')
+        c.argument('headers', nargs='+', help="Space separated custom headers in 'key[=value]' format that will be added to the webhook notifications. Use {} to clear existing headers.".format(quotes), validator=validate_headers)
+        c.argument('actions', nargs='+', help='Space separated list of actions that trigger the webhook to post notifications.', choices=['push', 'delete'])
+        c.argument('status', help='Indicates whether the webhook is enabled.', choices=['enabled', 'disabled'])
```

Comment:
This should probably be three_state_flag with enabled/disabled as the affirmative and negative labels.

Reply:
This would change the behavior?
I ported over the current behavior - https://github.com/Azure/azure-cli/blob/dev/src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/_params.py#L56.

If it would change things, I would want confirmation from @djyou first on this.

Reply:
Actually, looking at the parameter name, this is fine.  It wouldn't make sense to have `--status` as a flag. Disregard. :D